pkg/capabilities/targets/chainwriter/chainwriter_target-schema.json

```diff
@@ -18,14 +18,19 @@
           "type": "string",
           "enum": ["oneAtATime", "allAtOnce"],
           "description": "The schedule which must be the string 'oneAtATime'."
-        }
+        },
+	"cre_step_timeout": {
```

Comment:
This is breaking because it is a required property. 

Two thoughts on that:
1. Can we have a unit test showing that this is indeed breaking? Fail case for `testutils.AssertWorkflowSpec` without it in there.
2. Can we have a default to make it not breaking?

Reply:
Totally agree. we have spent days trying to figure out what was the culprit for failing e2e tests